plugins/polio/js/src/domains/Chronogram/ChronogramDetails/Modals/ChronogramTaskCreateEditModal.tsx

```diff
@@ -69,6 +71,9 @@ const CreateEditChronogramTaskModal: FunctionComponent<Props> = ({
     const allowConfirm =
         !formik.isSubmitting && formik.isValid && isFormChanged;
 
+    const currentUser = useCurrentUser();
+    const isUserAdmin = userHasPermission('iaso_polio_chronogram', currentUser);
```

Comment:
```suggestion
    const isUserAdmin = userHasPermission(Permission.POLIO_CHRONOGRAM, currentUser);
```
and import Permission import * as Permission from '../../../../../../../../hat/assets/js/apps/Iaso/utils/permissions';

Reply:
Fixed in https://github.com/BLSQ/iaso/pull/1530/commits/f0cc7762dfe8ba1d92621cee05a6bf1204ba74ad